docs/production-deployment/cloud/billing-and-cost.mdx

```diff
@@ -76,3 +78,19 @@ The Invoices table has the following information:
 - Balance due - Balance remaining after credits are granted
 
 You have the ability to download your Invoices by clicking the download icon by the date. 
+
+## Credit table (#credit-table}
```

Comment:
It's not building because that open parenthesis should be an open curly brace. This should fix it for you. 😺 

```suggestion
## Credit table {#credit-table}
```